packages/next/src/utils/config.ts

```diff
@@ -19,10 +21,11 @@ import { normalizeAssets } from '@nrwl/web/src/utils/normalize';
 import { createCopyPlugin } from '@nrwl/web/src/utils/config';
 import { WithNxOptions } from '../../plugins/with-nx';
 import {
-  computeCompilerOptionsPaths,
   createTmpTsConfig,
   DependentBuildableProjectNode,
 } from '@nrwl/workspace/src/utilities/buildable-libs-utils';
+import { importConfig } from './require-shim';
+const { default: loadConfig } = importConfig();
```

Comment:
Can just make this 

```
const loadConfig = importConfig();
```

and then in the shim file do

```

export function importConfig() {
  try {
    return require('next/dist/next-server/server/config').default;
  } catch {
    return require('next/dist/server/config').default;
  }
}
```

That way is is more consistent with how you'd  use default imports.